tsconfig.test.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Any reason why this is needed? Seems the tests work without it

Reply:
Mainly because without this I would have difference between what's VSCode considered like an error an the result I would get out of `type-check`. With this being explicitly set, I get the same consistent result between the VSCode editor and the type-check results when running the tests with `jest`.